apple_framework_relative_headers.bzl

```diff
@@ -15,7 +15,7 @@ def _apple_framework_relative_headers_impl(ctx):
     outputs.append(framework_header_file)
 
   include_dir = "/".join([
-      ctx.configuration.bin_dir.path, ctx.label.package, output_dir])
```

Comment:
Tip: Could you please add a comment documenting the data type of `ctx`? I know it's not part of this change, but since I have no idea what `ctx` is nor where its documentation lies, it's difficult for me to understand if the changes make sense.

Reply:
Added doc links.

Reply:
Thanks!